src/Lorentz/Contracts/RegistryDAO.hs

```diff
@@ -192,6 +233,8 @@ config = DAO.defaultConfig
 
   , DAO.cMaxVotes = 1000
   , DAO.cMaxProposals = 500
+
+  , DAO.cCustomCall = customEntrypointsHandler @_ @_ @k @v
```

Comment:
Nitpick, but given that this is only used here, why not move the type arguments in the `forall` to have `k` and `v` first?

Reply:
Yes, changed.